src/main/java/gim/logic/commands/SortCommand.java

```diff
@@ -0,0 +1,24 @@
+package gim.logic.commands;
+
+import static gim.model.Model.PREDICATE_SHOW_ALL_EXERCISES;
+import static java.util.Objects.requireNonNull;
+
+import gim.model.Model;
+
+/**
+ * Lists all exercises in the exercise tracker to the user.
```

Comment:
Could be more descriptive in what kind of list it provides.

Reply:
I agree, this needs to be changed. Thanks for spotting it!